src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/NPathComplexityCheck.java

```diff
@@ -102,21 +106,24 @@ public void setMax(int max) {
             TokenTypes.LITERAL_IF,
             TokenTypes.LITERAL_ELSE,
             TokenTypes.LITERAL_SWITCH,
-            TokenTypes.LITERAL_CASE,
-            TokenTypes.LITERAL_TRY,
-            TokenTypes.LITERAL_CATCH,
+            TokenTypes.CASE_GROUP,
             TokenTypes.QUESTION,
+            TokenTypes.LITERAL_RETURN,
+            TokenTypes.LITERAL_DEFAULT,
         };
     }
 
     @Override
     public int[] getRequiredTokens() {
-        return new int[] {
-            TokenTypes.CTOR_DEF,
-            TokenTypes.METHOD_DEF,
-            TokenTypes.INSTANCE_INIT,
-            TokenTypes.STATIC_INIT,
-        };
+        return getAcceptableTokens();
```

Comment:
that was in https://github.com/checkstyle/checkstyle/pull/2914#issuecomment-185746384

> I think we should call getAcceptableTokens from getRequiredTokens.